hangman.rb

```diff
@@ -0,0 +1,195 @@
+require 'colorize'
+
+class Hangman
+	def initialize
+		@words = ["method", "index", "bracket", "braces", "gem", "git", "terminal", "length", 
+							"count", "push", "class", "hash", "comment", "colorize", "mastermind", "orange", 
+							"online", "dragon", "seattle", "rubric"]
+		@word = @words.sample
+		@blank_spaces = "_ -" * @word.length
+		@blank_array = @blank_spaces.split("-")
+		@answer_letters = @word.split(//)
+		@wrong_answers = 0
+		@guess_array = []
+		@alphabet = %w(a b c d e f g h i j k l m n o p q r s t u v w x y z)
+		start
+	end
+
+	def start
+		puts """
+		Let's play Hangman!!!
+		Can you guess our Ada class related term?
+
+		|     _________
+		|     |/      |
+		|     |      
+		|     |      
+		|     |      
+		|     |      
+		|     |
+		| ____|___ 
+
+		word: #{@blank_array.join}
+		"""
+		get_guess
+	end
+
+# GET GUESSES ------------------------------------------------------------->
+	# Retrieve guess from user and allows them to exit the game by typing quit.
+	# Guesses are pushed to an array.
+	def get_guess
+		puts "Please type a letter to guess the word above."
+		puts "To exit the game, type 'quit'."
+		@guess = gets.chomp.downcase
```

Comment:
Based on the way you are using this `@guess` variable here it seems like it doesn't need to be an instance variable. You could successfully pass is in as a parameter to the guess_in_answer method.